tests/access_test.go

```diff
@@ -278,8 +279,9 @@ var _ = Describe("[rfe_id:500][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 				core.GroupName,
 				"virtualmachineinstancemigrations",
 				false,
-				allowAllFor("admin"),
-				denyDeleteCollectionFor("edit"),
+				allowAllFor("migrate"),
```

Comment:
Can we test for cluster-admin?

Reply:
`cluster-admin` is the super-user access role able to perform any action on any resource in any namespace and indeed is defined as:
```
rules:
- apiGroups:
  - '*'
  resources:
  - '*'
  verbs:
  - '*'
```
I don't think there is any need to test that.